src/frontend/src/tests/pages/WorkPackageDetailPage/WorkPackageViewContainer/WorkPackageDetails.test.tsx

```diff
@@ -3,82 +3,140 @@
-    expect(screen.getByText(`${wp.status}`, { exact: false })).toBeInTheDocument();
-    expect(screen.getByText(`${wp.name}`, { exact: false })).toBeInTheDocument();
-    expect(screen.getByText(`${fullNamePipe(wp.projectLead)}`, { exact: false })).toBeInTheDocument();
-    expect(screen.getByText(`${fullNamePipe(wp.projectManager)}`, { exact: false })).toBeInTheDocument();
-
-    expect(screen.getByText(`${weeksPipe(wp.duration)}`, { exact: false })).toBeInTheDocument();
-    expect(screen.getByText(`${wp.startDate.toLocaleDateString()}`, { exact: false })).toBeInTheDocument();
-    expect(screen.getByText(`${datePipe(wp.endDate)}`, { exact: false })).toBeInTheDocument();
-    const progresses = screen.getAllByText(`${percentPipe(wp.progress)}`); // progress and expectedProgress should be equal and return 2 results
-    expect(progresses.length).toBe(2);
-    expect(screen.getByText(`${wp.timelineStatus}`, { exact: false })).toBeInTheDocument();
+    it('renders work package manufacturing stage label', () => {
+      mockHook(false, false, users);
+      const { container } = renderComponent(exampleManufacturingWorkPackage);
+      const chips = Array.from(container.getElementsByClassName('MuiChip-label'));
+      expect(chips.length).toBe(2);
+      expect(chips[0]).toHaveTextContent('Manufacturing');
+    });
+
+    it('renders work package integration stage label', () => {
+      mockHook(false, false, users);
+      const { container } = renderComponent(exampleIntegrationWorkPackage);
+      const chips = Array.from(container.getElementsByClassName('MuiChip-label'));
+      expect(chips.length).toBe(2);
+      expect(chips[0]).toHaveTextContent('Integration');
+    });
   });
 
-  it('renders all the fields, example 3', () => {
-    mockHook(false, false, users);
-    const wp: WorkPackage = exampleWorkPackage3;
-    render(<WorkPackageDetails workPackage={wp} />);
-    expect(screen.getByText(`Work Package Details`)).toBeInTheDocument();
-    expect(screen.getByText(`${wp.status}`, { exact: false })).toBeInTheDocument();
-    expect(screen.getByText(`${wp.name}`, { exact: false })).toBeInTheDocument();
-    expect(screen.getByText(`${fullNamePipe(wp.projectLead)}`, { exact: false })).toBeInTheDocument();
-    expect(screen.getByText(`${fullNamePipe(wp.projectManager)}`, { exact: false })).toBeInTheDocument();
-    expect(screen.getByText(`${weeksPipe(wp.duration)}`, { exact: false })).toBeInTheDocument();
-    expect(screen.getByText(`${wp.startDate.toLocaleDateString()}`, { exact: false })).toBeInTheDocument();
-    expect(screen.getByText(`${datePipe(wp.endDate)}`, { exact: false })).toBeInTheDocument();
-    const progresses = screen.getAllByText(`${percentPipe(wp.progress)}`); // progress and expectedProgress should be equal and return 2 results
-    expect(progresses.length).toBe(2);
-    expect(screen.getByText(`${wp.timelineStatus}`, { exact: false })).toBeInTheDocument();
+  describe.skip('WP Fields', () => {
```

Comment:
idrc about these kinds of "render everything" tests so if we're have to skip them we may as well just delete them

Reply:
fixed the tests